my-sms/db/migrate/20210212025754_move_student_title_str_to_title_model.rb

```diff
@@ -0,0 +1,44 @@
+class MoveStudentTitleStrToTitleModel < ActiveRecord::Migration
+  def up
+    # cache titles from db to avoid multiple queries
+    title_name_to_id = {}
+    Title.all.each do |title|
+      title_name_to_id[title.name] = title.id
+    end
```

Comment:
This doesn't appear to be used anywhere, can remove

Reply:
Fixed this in dc0b638c58afe8b0c687e17b0e8882dd74c29a29